plutus-ledger-api/src/PlutusLedgerApi/V1/Value.hs

```diff
@@ -8,26 +10,24 @@
 {-# LANGUAGE OverloadedStrings    #-}
 {-# LANGUAGE TemplateHaskell      #-}
 {-# LANGUAGE TypeApplications     #-}
+{-# LANGUAGE TypeFamilies         #-}
+{-# LANGUAGE UndecidableInstances #-}
 {-# LANGUAGE ViewPatterns         #-}
-
--- Prevent unboxing, which the plugin can't deal with
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# OPTIONS_GHC -fexpose-all-unfoldings #-}
 {-# OPTIONS_GHC -fno-omit-interface-pragmas #-}
 {-# OPTIONS_GHC -fno-spec-constr #-}
 {-# OPTIONS_GHC -fno-specialise #-}
-{-# OPTIONS_GHC -fexpose-all-unfoldings #-}
-{-# LANGUAGE BlockArguments       #-}
-{-# LANGUAGE DataKinds            #-}
-{-# LANGUAGE TypeFamilies         #-}
-{-# LANGUAGE UndecidableInstances #-}
--- We need -fexpose-all-unfoldings to compile the Marlowe validator
```

Comment:
This comment should be preserved

Reply:
My logic was this:
In the Plutus documentation we have a list of GHC options that should be enabled for Plugin to work. Many Plinth modules use them. We should use the recommended options everywhere and there is nothing that makes this particular instance special enough to require and extra comment.

Reply:
`-fexpose-all-unfoldings` is not an option that should be enabled everywhere, as explained in https://plutus.cardano.intersectmbo.org/docs/using-plutus-tx/extensions-flags-pragmas

Reply:
Ok, I made a mistake assuming that this option should be used everywhere in Plinth by default.

Reply:
Do you know why `{-# INLINEABLE #-}` is not enough for the Marlowe validator?

Reply:
Missing unfolding for something, I think - we never identified the root cause.